### PR TITLE
Adds docs to `errTimelineItemValidation`

### PR DESCRIPTION
**Pull request by chris-ramon**
#### Details

- [x] Adds missing documentation to un-exported error `errTimelineItemValidation`.

### CODE REVIEW

File: traceapp/vis.go [hunk 1]
```diff
@@ -14,6 +14,8 @@ import (
 	_ "sourcegraph.com/sourcegraph/appdash/sqltrace"
 )
 
+// ErrTimelineItemValidation is returned by timelineItem.Valid when either
+// timelineItem.Label or timelime.ItemFullLabel are empty.
 var ErrTimelineItemValidation = errors.New("timeline item validation error")
```
Comment (slimsag) [inline traceapp/vis.go]: Thanks for sending this @chris-ramon !

Does this variable need to be exported at all? It seems like it is only used internally. Could it just be `errTimelineItemValidation` instead?

Comment (shurcooL) [inline traceapp/vis.go]: Agreed, if it doesn't need to be exported, it's better to make it not exported. It makes the public API more readable by having fewer entries.

Comment (chris-ramon) [inline traceapp/vis.go]: Totally agree as well, addressed on https://github.com/chris-ramon/appdash/commit/a1846dd95511253cac8038357b69b2f50daba985

Thanks for reviewing this guys :+1:

**Comment by shurcooL** [comment]
LGTM.

**Comment by slimsag** [comment]
Thanks! LGTM

**Pull request merged**